src/main/scala/temple/generate/kube/KubernetesGenerator.scala

```diff
@@ -181,6 +182,14 @@ object KubernetesGenerator {
     )
   }
 
+  private def buildKongFiles(): Map[File, FileContent] = Map(
```

Comment:
This shouldn’t have empty parens (as no side effects); it feels like it should be a `val` as it is just a static value